app/domain/authentication/authn_jwt/orchestrate_authentication.rb

```diff
@@ -7,13 +7,14 @@ module AuthnJwt
     OrchestrateAuthentication ||= CommandClass.new(
       dependencies: {
         validate_uri_based_parameters: Authentication::AuthnJwt::InputValidation::ValidateUriBasedParameters.new,
-        configuration_factory: Authentication::AuthnJwt::VendorConfigurations::ConfigurationFactory.new,
+        configuration_factory: Authentication::AuthnJwt::VendorConfigurations::CreateVendorConfiguration.new,
         jwt_authenticator: Authentication::AuthnJwt::Authenticator.new,
         logger: Rails.logger,
-        enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str
+        installed_authenticators_class: Authentication::InstalledAuthenticators
```

Comment:
why would you do that? in other cases we just pass the `enabled_authenticators` as an input